README.md

```diff
@@ -97,3 +97,19 @@ In order to generate the doxygen documentation for OSI, please follow the follow
 cmake -DFILTER_PROTO2CPP_PY_PATH=<path-to-proto2cpp.py> <path-to-CMakeLists.txt>
 ```
 5. The build process will then generate the doxygen documentation under the directory doc.
+
+
+Citing
+-------------
+
+For refering to OSI in your publication, please cite as:
```

Comment:
Use the following citation for referencing the OSI interface in your scientific work:

@martin-fzd  better?